source/code/plugins/in_sudo_tail.rb

```diff
@@ -58,7 +58,7 @@ def configure(conf)
  
       @parser = Plugin.new_parser(conf['format'])
       @parser.configure(conf)
-
+      @path.gsub!("," , " ")       #in case multiple paths are separated by commas
```

Comment:
edge case: the file path can contain , or space. although they are really rare.
do we allow wild card in the path? /etc/*.log